verl/trainer/config/rollout/rollout.yaml

```diff
@@ -1,12 +1,16 @@
 # Target class for this configuration
 _target_: verl.workers.config.RolloutConfig
 
-# actor_rollout_ref.rollout.name: hf/vllm/sglang. The default value will be removed in the future
+# actor_rollout_ref.rollout.name: hf/vllm/sglang/trtllm. The default value will be removed in the future
 name: ???
 
 # sync: LLM, async: AsyncLLM
 mode: async
 
+resource_pool_id: global_pool
+
+colocate_slot: 0
+
```

Comment:
Please review the core logic here. 

The global_pool is the actor policy pool by default. If Rollout is in `global_pool`, it is either colocated or hybrid to the actor; if it is not in `global_pool`, it is standalone, or colocated with other roles.